### PR TITLE
Feat: header page

### LINKED ISSUE #4: Home Page > Header
Inside `Pages` create a new folder `HomePage` or `LandingPage` and create a new folder `Header` and inside it `Header.js`

image

### PR DESCRIPTION
**Pull request by Abdellatif-E**

### CODE REVIEW

File: src/Pages/HomePage/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import Button from '../../../Components/button/button';
+
+function Header() {
+  return (
+    <div className="bg-gray-100 min-h-screen">
+      <img src={Logo} className="w-1/12 h-auto absolute left-14 top-9" alt="" />
+      <img
+        src={img1}
+        className=" absolute top-0 right-0 w-6/12"
+        alt=""
```
Comment (osamaakb) [inline src/Pages/HomePage/Header/Header.js]: added alt text here

File: src/Pages/HomePage/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <div className="w-52 h-14 absolute top-5 right-5">
+        <Button text="Get Started" />
+      </div>
+
+      <div className="absolute top-1/3 right-2/4 h-60">
+        <div className="leading-10 font-bold md:text-center ">
+          <span className="text-5xl ">
+            Your life is your biggest{' '}
+            <span className="text-primary"> Project </span>{' '}
+          </span>{' '}
```
Comment (osamaakb) [inline src/Pages/HomePage/Header/Header.js]: remove these `{'  '}`

File: .eslintcache [hunk 3]
```diff
@@ -1 +1 @@
...
...op\\2020 Re-Coded\\Capstone project\\e-track-web\\src\\App.js":"2","D:\\uni\\workshop\\2020 Re-Coded\\Capstone project\\e-track-web\\src\\reportWebVitals.js":"3"},{"size":517,"mtime":1612984958064,"results":"4","hashOfConfig":"5"},{"size":551,"mtime":1612992908583,"results":"6","hashOfConfig":"5"},{"size":375,"mtime":1612984958064,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"1upg5ur",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"D:\\uni\\workshop\\2020 Re-Coded\\Capstone project\\e-track-web\\src\\index.js",[],["15","16"],"D:\\uni\\workshop\\2020 Re-Coded\\Capstone project\\e-track-web\\src\\App.js",[],"D:\\uni\\workshop\\2020 Re-Coded\\Capstone project\\e-track-web\\src\\reportWebVitals.js",[],{"ruleId":"17","replacedBy":"18"},{"ruleId":"19","replacedBy":"20"},"no-native-reassign",["21"],"no-negated-in-lhs",["22"],"no-global-assign","no-unsafe-negation"]
```
Comment (osamaakb) [inline .eslintcache]: don't commit this file

**Review by osamaakb — CHANGES_REQUESTED** [review]
Good Job abdellatif, the about page is being sent with your PR cause of your code formatted is changing something there so maybe don't include it in your commit too + check the comments too

**Comment by Abdellatif-E** [comment]
All done i added the alt text , removed the unneeded symbols also i did git fetch and pull so it's up-to-date
thank you osama for feedback

File: .eslintcache (hunk 3 above)
Comment (osamaakb) [inline .eslintcache]: can you not commit this one?

File: src/App.js [hunk 4]
```diff
@@ -1,6 +1,12 @@
 import './App.css';
+import Header from './Pages/HomePage/Header/Header';
+
 function App() {
```
Comment (osamaakb) [inline src/App.js:1]: make your app.js similar to the one on master

**Review by osamaakb — CHANGES_REQUESTED** [review]

**Review by osamaakb — APPROVED** [review]

**Pull request merged**